bundle/manifests/apicast-operator.clusterserviceversion.yaml

```diff
@@ -103,6 +103,11 @@ spec:
           template:
             metadata:
               labels:
+                com.redhat.component-name: apicast-operator
+                com.redhat.component-type: infrastructure
+                com.redhat.component-version: 0.6.0
+                com.redhat.product-name: 3scale
```

Comment:
Some thoughts about this:
* Should in this case be product a different than 3scale? APIcast?

Reply:
I would say it is 3scale. It is productized with 3scale.

@thomasmaas might have a better answer